e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -175,24 +168,14 @@ describe("scenarios > setup", () => {
     cy.visit("/");
 
     cy.location("pathname").should("eq", "/setup");
-    cy.findByTestId("welcome-page").within(() => {
-      cy.findByText("Welcome to Metabase");
-      cy.findByTextEnsureVisible("Let's get started").click();
-    });
+
+    skipWelcomePage();
 
     cy.findByTestId("setup-forms").within(() => {
-      // Language
-      cy.findByText("What's your preferred language?");
-      cy.findByText("English").click();
-      cy.button("Next").click();
+      selectPreferredLanguageAndContinue();
 
       // User
-      cy.findByText("What should we call you?");
-      cy.findByLabelText("Email").type(admin.email);
-      cy.findByLabelText("Company or team name").type("Epic Team");
-      cy.findByLabelText("Create a password").type(admin.password);
-      cy.findByLabelText("Confirm your password").type(admin.password);
-      cy.button("Next").click();
+      fillUserAndContinue({ firstName: null, lastName: null });
```

Comment:
I'd like to see we require an argument for this util so we know what'll be set instead of having to go into the implementation detail to see that the default value is from an admin.

I think this would be easier to understand

```suggestion
      fillUserAndContinue({...admin, firstName: null, lastName: null});
```

We could probably do this for line 51 and below as well. There is `USERS.normal` maybe that will also work, otherwise, we could just create a `user` object which contains the information from line 51 and below. So we could use the same util
```ts
      fillUserAndContinue(normal);
```

Reply:
Made all arguments non required so it's easier to read. e17682641defb9c484ef0a3fa892116962d39983
I left the test at line ~50 as it is as that was testing weak and strong password validation.